index.html

```diff
@@ -83,5 +123,60 @@ <h2>Headliners</h2>
             </div>
         </section>
     </main>
+
+    <section id="about" class="unactive">
+        <div id="about-header">
+            <h2 id="about-title">All Points East <br />Fest 2023</h2>
+            <hr />
+            <p id="about-text">
+                The first year of All Points East was held in 2018 with LCD Soundsystem,
+                The xx and Björk headlining the festival weekend and Catfish and the Bottlemen,
+                The National and Nick Cave headlining the three standalone shows the following weekend.
+            </p>
+            <p id="about-contact">
+                Please contact us per Email for nay further questions about
+                All Points East 2023!<br />
+                <address>allpointseast2023@gmail.com</address>
+            </p>
+        </div>
+
+        <div id="see-past">
+            <h2>See the past All Points East Festivals</h2>
+            <hr />
+            <p>Take a look at the past two All Points East Festivals</p>
+            <div id="past-ctr">
+                <div class="past-event-card">
+                    <div class="overlay"></div>
+                    <h3>2019</h3>
+                </div>
+                <div class="past-event-card">
+                    <div class="overlay"></div>
+                    <h3>2021</h3>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <footer>
+        <div id="partners-ctr">
+            <h2>Partners</h2>
+            <hr />
+            <div id="partners">
+                <img src="./images/partners/mozilla.svg" />
+                <img src="./images/partners/googlekorea.svg" />
+                <img src="./images/partners/naver.svg" />
+                <img src="./images/partners/daumkakaologo.svg" />
+                <img src="./images/partners/airbnb.svg" />
+            </div>
+        </div>
+        <div id="inner-footer">
+            <img src="./images/logo.jpg" />
+            <small>
+                2023 All East Points.<br/>
+                Some rights reserved.
+            </small>
+        </div>
```

Comment:
- [ ] The footer on the original design only has dark background color while your design carries white background color.
- [ ] Also, there's no partner section on the desktop version of the original design except on the mobile version. Kindly remove that part.

### Your Design
![your-about-footer](https://github.com/Movgto/capstone_1_project/assets/26597432/491eaaf7-a3a1-4d79-b274-48d02dc7ab3e)


### Template
![temp-about-footer](https://github.com/Movgto/capstone_1_project/assets/26597432/45ea5c99-534f-4f47-8e26-e1638606a160)